src/uct/sm/self/self.c

```diff
@@ -68,8 +68,7 @@ static ucs_status_t uct_self_iface_query(uct_iface_h tl_iface, uct_iface_attr_t
                                    UCT_IFACE_FLAG_GET_BCOPY        |
                                    UCT_IFACE_FLAG_ATOMIC_CPU       |
                                    UCT_IFACE_FLAG_PENDING          |
-                                   UCT_IFACE_FLAG_CB_SYNC          |
-                                   UCT_IFACE_FLAG_EP_CHECK;
```

Comment:
maybe no need to remove, but initialize empty function which returns ok?
this way self transport can be selected for the cases when EP_CHECK is requested

Reply:
test keepalive on itself? why? it will add ops to keepalive loop

Reply:
because you will definitely notice if something happened with "peer", but if it is not supported, self transport will never be selected when keepalive is requested

Reply:
ok, restored